extensions/amp-nested-menu/0.1/amp-nested-menu.js

```diff
@@ -314,14 +316,25 @@ export class AmpNestedMenu extends AMP.BaseElement {
     if (!item) {
       return;
     }
-    const nextItem =
-      e.key == Keys.UP_ARROW
+    let nextItem =
+      e.key == Keys.UP_ARROW || e.key == Keys.HOME
         ? item.previousElementSibling
         : item.nextElementSibling;
+
     // have reached the beginning or end of the list.
     if (!nextItem) {
       return;
     }
+
+    // Handle HOME key (selects first item in the list)
```

Comment:
Why iterate through the siblings in a `while` loop rather than grab the parent element's `firstChild`?